releasenotes/notes/feature-mimo-59c3d6d02335061c.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Add ``mimo()`` function for generating a multi-input 
+    multiple-output (MIMO) channel-decoding problem. 
+  - |  
+    Add ``comp()`` function for generating a coordinated multi-point 
```

Comment:
Need to update the release note to match the new function name.

Reply:
I felt I was missing something. Maybe this next commit will trigger CI, which seems to have missed [the previous commit](https://github.com/dwavesystems/dimod/pull/1357/commits/b00e943aad189ff1590d225877611c06f071d0cd)

Reply:
Updated with [this commit](https://github.com/dwavesystems/dimod/pull/1357/commits/7edb35e3197ea3d1a5ba5d25ba6aeb0a3f10b7bf)